src/handlers/cars/getCarList.spec.ts

```diff
@@ -0,0 +1,12 @@
+import axios from 'axios'
+import { API_URL } from '../../constants/http-constants';
+
+describe('getCarList', () => {
+    describe('When getCarList had been called', () => {
+        test('getCarById should return 200 OK', () => {
+            axios.get(`${API_URL}/cars`).then((response) => {
```

Comment:
Viktar, I would recommend you setup prettier in order to keep code formatting consistent.
Also, unit tests should not make real API calls. When you cal real API calls it looks more like integration testing.
Your test cases should call lambda function directly, pass mock event to it and check results ;)

Reply:
The CORS error showed because I trying to set up the dynamoDb for task4 , everything worked before I've started task4